sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -572,6 +572,9 @@ public static void SendEvents_TestHub(int numEvents, string input, [EventHub(Tes
                     Assert.AreEqual(s_processedEventCount++, propertiesArray[i]["TestIndex"]);
                 }
 
+                Assert.NotNull(partitionContext.PartitionId);
+                Assert.NotNull(partitionContext.ReadLastEnqueuedEventProperties());
```

Comment:
`LastEnqueuedEventProperties` is a `struct`, so never null.    Maybe compare to `default`?